src/components/HomePageItemCard/HomePageItemCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Card } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import coffeepot from "../../images/coffeepot.jpg";
+import "./HomePageItemCard.css";
+
+function HomePageItemCard({ itemName = "", id }) {
+  return (
+    <Card className="homePageItemCard">
+      <Card.Img className="itemImage mx-auto" variant="top" src={coffeepot} />
```

Comment:
I think here in the src it should be coming from the props too in case you have different items with different images, and you can also have a deafult image in case no image is passed

Reply:
You're totally right!

By the way, I get a warning like this? Do you have any suggestions? I tried to read the doc and validate the props, but couldn't do it.
![image](https://user-images.githubusercontent.com/81304552/129013476-b9a4307a-e48c-4556-a14b-7009f4e6002d.png)